aws_database/db_service/models.py

```diff
@@ -0,0 +1,177 @@
+    iso639 = Column(Text, unique=True)
+    name = Column(Text)
+
+class IsoScript(Base):
+    __tablename__ = 'isoScript'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    iso15924 = Column(Text, unique=True)
+    name = Column(Text)
+
+class LocalizedBookName(Base):
+    __tablename__ = 'localizedBookName'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(Text)
+    bookreference_id = Column(Integer, ForeignKey('bookReference.number'))
+    bibleversion_id = Column(Integer, ForeignKey('bibleVersion.id'))
+
+class Question(Base):
+    __tablename__ = 'question'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    start_verse = Column(Text, ForeignKey('verseReference.fullverseid'))
+    end_verse = Column(Text, ForeignKey('verseReference.fullverseid'))
+    language = Column(Text, ForeignKey('isoLanguage.iso639'))
+    question = Column(Text)
+    answer = Column(Text)
+
+class VerseReference(Base):
+    __tablename__ = 'verseReference'
+    
+    fullverseid = Column(Text, primary_key=True, index=True)
+    number = Column(Integer)
+    chapter = Column(Text, ForeignKey('chapterReference.fullchapterid'))
+    bookreference = Column(Text, ForeignKey('bookReference.abbreviation'))
+
+class VerseText(Base):
+    __tablename__ = 'verseText'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    text = Column(Text)
+    biblerevision_id = Column(Integer, ForeignKey('bibleRevision.id'))
+    versereference = Column(Text, ForeignKey('verseReference.fullverseid'))
+    book = Column(Text)
+    chapter = Column(Integer)
+    verse = Column(Integer)
+
+class AssessmentMissingWords(Base):
+    __tablename__ = 'assessmentMissingWords'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    assessment_id = Column(Integer, ForeignKey('assessment.id'), nullable=False)
+    score = Column(Numeric)
+    flag = Column(Boolean)
+    vref = Column(Text, ForeignKey('verseReference.fullverseid'))
+    source = Column(Text)
+    note = Column(Text)
+    target = Column(JSONB)
+
+class AlignmentThresholdScores(Base):
+    __tablename__ = 'alignmentThresholdScores'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    assessment_id = Column(Integer, ForeignKey('assessment.id'), nullable=False)
+    score = Column(DOUBLE_PRECISION)
+    flag = Column(Boolean)
+    note = Column(Text)
+    vref = Column(Text)
+    source = Column(Text)
+    target = Column(Text)
+    hide = Column(Boolean)
+
+class AlignmentTopSourceScores(Base):
+    __tablename__ = 'alignmentTopSourceScores'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    assessment_id = Column(Integer, ForeignKey('assessment.id'), nullable=False)
+    score = Column(DOUBLE_PRECISION)
+    flag = Column(Boolean)
+    note = Column(Text)
+    vref = Column(Text)
+    source = Column(Text)
+    target = Column(Text)
+    hide = Column(Boolean)
+
+class GroupResultsChapter(Base):
```

Comment:
I think this can be removed. It was a view, but I've switched to doing an on-the-fly table join, which I think was just as fast.

Reply:
tables dropped!